config/request.lua

```diff
@@ -0,0 +1,72 @@
+if lighty.env["request.method"] == "GET" then
+
+    -- obtain service type from environment
+    serviceType = os.getenv('SERVICE_TYPE')
+    if serviceType == nil then
+        print('SERVICE_TYPE environment variable is missing')
+        return 500
+    end
+
+    serviceType = serviceType:lower()
+
+    path = lighty.env["uri.path"]
+    query = lighty.env["uri.query"]
+
+
+    -- check if query is present
+    if not query then
+        return 400
+    end
+
+    -- redirect to MapServer
+    if path ~= "/mapserver" then
+        lighty.env["request.uri"] = "/mapserver?" .. query
+        return lighty.RESTART_REQUEST
+    end
+
+    -- parse and rewrite query string
+    params = {}
+    newQuery = ''
+    for k, v in query:gmatch("([^?&=]+)=([^&]+)") do
+        k = k:lower()
+
+        params[k] = v
+
+        -- remove map parameter from query
+        if k ~= 'map' then
```

Comment:
Het verwijderen van de map query parameter werd voorheen ook gedaan voor een POST request. Ik denk dat je dit hele "parse and rewrite query string" block het beste buiten de "method" if kan doen.